ui/raidboss/timeline.ts

```diff
@@ -1086,14 +1086,14 @@ export class TimelineController {
   constructor(
     protected options: RaidbossOptions,
     protected ui: TimelineUI,
-    raidbossDataFiles: { [filename: string]: string },
+    raidbossDataFiles: { [filename: string]: string | LooseTriggerSet },
```

Comment:
Can this be `raidbossDataFiles: { [filename: string]: RaidbossFileData },` to avoid duplicating the definition of `RaidbossFileData`? (Maybe that means `trigger.d.ts` has to export the `RaidbossFileData` type?)

Reply:
True, I think to export `RaidbossFileData` type is better.